packages/monorepo-scripts/src/deps_versions.ts

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import * as fs from 'fs';
+import {sync as globSync} from 'glob';
+import * as _ from 'lodash';
+
+interface Dependencies {
+    [depName: string]: string;
+}
+interface Versions {
+    [packageName: string]: string;
+}
+interface VersionsByDependency {
+    [depName: string]: Versions;
+}
+
+function log(...args: any[]) {
+    console.log(...args); // tslint:disable-line:no-console
+}
+
+function getDependencies(path: string): Dependencies {
+    const file = fs.readFileSync(path).toString();
+    const parsed = JSON.parse(file);
+    const dependencies = {
+        ...parsed.dependencies,
+        ...parsed.devDependencies,
+    };
+    return dependencies;
+}
+
+function getPackageName(path: string): string {
```

Comment:
Let's not make this a stand alone function since it is only used in one place and doesn't generalize well.